go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -415,23 +391,26 @@ func (oc *Controller) createEgressFirewallRules(priority int, match, action, ext
 	if err != nil {
 		return fmt.Errorf("error executing find ACL command, stderr: %q, %+v", stderr, err)
 	}
+	sort.Strings(logicalSwitches)
```

Comment:
this is used to ensure the fexec match works in unit tests and isnt really required for the code to function. We should add a comment to remove this in the future, but wont block merge on it